### PR TITLE
Some fireaxe love

### PR DESCRIPTION
**Pull request by Alliostra**
## About The Pull Request

The Fire axe has received a small damage and penetration buff (from 33 damage wielded to 34 damage wielded and from 1.6 AP to 1.8 AP), since it was pretty shit for a weapon that doesn't fit into a bag.

The fire axe cabinet can now also be unlocked by an ID with head or medical access without smashing the glass.

## Why It's Good For The Game

No longer neglected

## Changelog
:cl:
add: the fire axe cabinet can now be opened by an ID with head or medical access.
balance: slightly buffed fire axe damage
/:cl:

### CODE REVIEW

**Comment by DiaFRAME444** [comment]
Should the Fireaxe really be so strong? I'm pretty sure its better than the NT Greatsword now.

**Comment by assortedbeads** [comment]
Axes do tend to be stronger than swords

**Comment by Alliostra** [comment]
> Should the Fireaxe really be so strong? I'm pretty sure its better than the NT Greatsword now.

The Fireaxe deals 2 less damage than the greatsword (both wielded), and has slightly worse AP too. It also occupies the slot usually reserved for a big gun, and deals a puny 10 damage when not wielded.

**Comment by DiaFRAME444** [comment]
> The Fireaxe deals 2 less damage than the greatsword (both wielded)

The way the multiplier works means a massive damage jump when a Spike toolmod is added. Still wasn't a top tier weapon just yet at 3.3x, but 3.7x and AP buff might make it so.
I guess a more fair comparsion would be the Halberda.

**Comment by Alliostra** [comment]
> > The Fireaxe deals 2 less damage than the greatsword (both wielded)
> 
> The way the multiplier works means a massive damage jump when a Spike toolmod is added. Still wasn't a top tier weapon just yet at 3.3x, but 3.7x and AP buff might make it so. I guess a more fair comparsion would be the Halberda.

Good point.

File: code/game/objects/structures/fireaxe_cabinet.dm [hunk 1]
```diff
@@ -10,6 +10,7 @@
 	var/unlocked
 	var/shattered
 	var/obj/item/tool/fireaxe/fireaxe
+	req_one_access = list(access_moebius, access_heads)
```
Comment (Firefox13) [inline code/game/objects/structures/fireaxe_cabinet.dm:13]: I remember seeing a fire axe in technomancers locker room too

**Comment by Pink-Chink** [comment]
image

**Review by Pink-Chink — APPROVED** [review]

**Pull request merged**